### PR TITLE
linkage_arm.S: Fix VFP SOFTFP target

### PR DESCRIPTION
**Pull request by gizmo98**
https://github.com/mupen64plus/mupen64plus-core/issues/72

### CODE REVIEW

File: src/r4300/new_dynarec/linkage_arm.S [hunk 1]
```diff
@@ -19,7 +19,7 @@
  * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
 	.cpu arm9tdmi
 #ifndef __ARM_NEON__
-#if (defined(__VFP_FP__) && !defined(__SOFTFP__))
+#if (defined(__VFP_FP__) && !defined(__SOFTFP__) && defined(__ARM_PCS_VFP))
```
Comment (littleguy77) [inline src/r4300/new_dynarec/linkage_arm.S]: Thanks for catching this one too :)

**Comment by littleguy77** [comment]
Thanks @gizmo98 !

@richard42 @Narann This fixes #72.

**Comment by Narann** [comment]
gcc specific macro? Mmmh, I will let @richard42 review this. It's typically the kind of thing which make code maintenance complex.

**Comment by littleguy77** [comment]
If you take issue with the use of GCC macros, then PR #67 should have been rejected.  This PR simply corrects the usage of the GCC macros introduced in PR #67.

This is a regression fix.  Without it, mupen64plus-ae cannot be built.

**Comment by littleguy77** [comment]
Thanks much :)

**Pull request merged**